src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -162,7 +167,7 @@ private async Task<long> GenerateArchive(MigrateRepoCommandArgs args)
             throw new OctoshiftCliException($"Bitbucket export failed --> State: {exportState}; Message: {exportMessage}");
         }
 
-        _log.LogInformation($"Export completed. Your migration archive should be ready on your instance at $BITBUCKET_SHARED_HOME/data/migration/export/Bitbucket_export_{exportId}.tar");
+        _log.LogInformation($"Export completed. Your migration archive should be ready **on your Bitbucket instance** at $BITBUCKET_SHARED_HOME/data/migration/export/Bitbucket_export_{exportId}.tar");
```

Comment:
Love the addition of this language!

I was curious about the link provided when I first saw it printed to the console because it's clickable but doesn't (ofc) take you anywhere. Is the migration available to see in the BBS UI at all?

Reply:
Afraid not! :(

Reply:
Is there a reason we share this with folks then?

Reply:
We have this output because, depending on your arguments, the CLI might stop here and expect you to go to the Bitbucket instance manually to grab the archive. This tells you where the archive will be.